Library/Homebrew/test/.rubocop_todo.yml

```diff
@@ -121,17 +121,6 @@ RSpec/RepeatedDescription:
     - 'rubocops/lines_cop_spec.rb'
     - 'tab_spec.rb'
 
-# Offense count: 10
-RSpec/RepeatedExample:
```

Comment:
Are these examples with exactly the same code or just the same `it`/name?

Reply:
These are not repeated, they are using a different setup, so I'm not sure what this is flagging.

Reply:
Not the same `it` string?

Reply:
I don't think so. For example, one has “Sh“ and one has ”Bash“ at the end.

Reply:
Oh, is is indeed the same exact code as the one before …

Reply:
Yes, unless I've misunderstood something these should all be duplicate tests that I've removed.

Reply:
The code is (or was) duplicated with different `it` strings.

Reply:
You haven't misunderstood, you're correct 👍